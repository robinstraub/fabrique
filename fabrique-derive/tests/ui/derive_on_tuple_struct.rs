use fabrique_derive::Factory;

#[derive(Factory)]
struct Anvil(u32, u32);

fn main() {}