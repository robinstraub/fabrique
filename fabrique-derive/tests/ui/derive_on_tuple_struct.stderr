error: Factory can only be derived from named structs, tuple struct given
 --> tests/ui/derive_on_tuple_struct.rs:4:1
  |
4 | struct Anvil(u32, u32);
  | ^^^^^^
